src/main/kotlin/com/workos/mfa/MfaApi.kt

```diff
@@ -174,12 +175,13 @@ class MfaApi(private val workos: WorkOS) {
       .data(challengeFactorOptions)
       .build()
 
-    return workos.post("/auth/factors/challenge", Challenge::class.java, config)
+    return workos.post("/auth/factors/${challengeFactorOptions.authenticationFactorId}/challenge", Challenge::class.java, config)
```

Comment:
We should also update the `ChallengeFactorOptions` so it does not send the `authenticationFactorId` in the request body.